ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,658 @@
+            breadcrumbsList.length = lastVisibleLevel + 2;
+        }
+
+        if (breadcrumbsOptions.position && breadcrumbsOptions.position.align !== 'left') {
+            breadcrumbs.alignGroup();
+        }
+    }
+
+    /**
+     * Create a group, then draw breadcrumbs together with the separators.
+     *
+     * @requires  modules/drilldown
+     *
+     * @function Highcharts.Chart#draw
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     */
+    public draw(this: Breadcrumbs): void {
+        const breadcrumbs = this,
+            chart = breadcrumbs.chart,
+            breadcrumbsList = breadcrumbs.breadcrumbsList,
+            breadcrumbsOptions = chart.options.drilldown?.breadcrumbs,
+            lastBreadcrumbs = breadcrumbsList[breadcrumbsList.length - 2][2],
+            buttonPadding = 5;
+
+        // A main group for the breadcrumbs.
+        if (!this.breadcrumbsGroup) {
+            this.breadcrumbsGroup = chart.renderer
+                .g('breadcrumbs-group')
+                .attr({
+                    zIndex: 7
+                })
+                .addClass('highcharts-drilldown-breadcrumbs')
+                .add();
+        }
+
+        // Inital position for calculating the breadcrumbsGroup.
+        let posX: number = lastBreadcrumbs ? lastBreadcrumbs.x +
+                lastBreadcrumbs.element.getBBox().width + buttonPadding : 0;
+        const posY: number = buttonPadding;
+
+        // Draw breadcrumbs.
+        if (breadcrumbsList && breadcrumbsOptions) {
+            breadcrumbsList.forEach(function (breadcrumb, index): void {
+                const breadcrumbButton: SVGElement = breadcrumb[2];
+
+                // Render a button.
+                if (!breadcrumbButton) {
+                    const button = breadcrumbs.renderButton(breadcrumb, posX, posY);
+                    breadcrumb.push(button);
+                    posX += button ? button.getBBox().width + buttonPadding : 0;
+                    breadcrumbs.breadcrumbsGroup.lastXPos = posX;
+                }
+
+                // Render a separator.
+                if (index === breadcrumbsList.length - 2) {
+                    const separator = breadcrumbs.renderSeparator(breadcrumb, posX, posY);
+
+                    breadcrumb.push(separator);
+                    posX += separator ? separator.getBBox().width + buttonPadding : 0;
+                }
+            });
+        }
+        breadcrumbs.alignGroup();
+    }
+
+    /**
+     * Performs multiple drillups based on the provided number.
+     *
+     * @requires  modules/drilldown
+     *
+     * @function Highcharts.Chart#multipleDrillUp
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     * @param {Highcharts.Breadcrumbs} drillAmount
+     *        A number of drillups that needs to be performed.
+     */
+    public multipleDrillUp(this: Breadcrumbs, drillAmount: number): void {
```

Comment:
If this method should stay, it should be moved outside of Breadcrumbs class

Reply:
Do you mean into `Chart`?